cdap-app-fabric/src/test/java/io/cdap/cdap/DummyAppWithTrackingTable.java

```diff
@@ -49,6 +49,10 @@
  */
 @SuppressWarnings("unused")
 public class DummyAppWithTrackingTable extends AbstractApplication {
+  
+  public String getName() {
```

Comment:
Why adding this?

Reply:
must have added it for some tests that were previously not ignored - removing it